oam/etc/Columnstore.xml

```diff
@@ -247,7 +247,7 @@
         <DBRoot1>/var/lib/columnstore/data1</DBRoot1>
         <DBRMRoot>/var/lib/columnstore/data1/systemFiles/dbrm/BRM_saves</DBRMRoot>
         <TableLockSaveFile>/var/lib/columnstore/data1/systemFiles/dbrm/tablelocks</TableLockSaveFile>
-		<DBRMTimeOut>20</DBRMTimeOut> <!-- in seconds -->
+		<DBRMTimeOut>15</DBRMTimeOut> <!-- in seconds -->
```

Comment:
Wow, didn't realize we've reduced this so much.  This might be dangerous; with a large extent map, an op can legitimately take a long time.  AFAIK, that's still true, but maybe that has been changed too, you guys will have to tell me on this one.

Reply:
Side note, for the 10.5 integration you'll have to be sure to include these files in the packing list for deb pkgs, if that's still a thing in 10.5.